From b9190f5d2e0198104f805dae14ae3b1f5ed9a5bd Mon Sep 17 00:00:00 2001
From: Ramid Khan <ramidzkh@gmail.com>
Date: Fri, 8 May 2020 20:10:45 +1000
Subject: [PATCH] Fix 1.15.2 mappings

---
 mappings/bukkit-1.15.2-members.csrg | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/mappings/bukkit-1.15.2-members.csrg b/mappings/bukkit-1.15.2-members.csrg
index 01ea590..d04f0f4 100644
--- a/mappings/bukkit-1.15.2-members.csrg
+++ b/mappings/bukkit-1.15.2-members.csrg
@@ -1282,13 +1282,13 @@ BossBattleCustomData a (LMinecraftKey;LIChatBaseComponent;)LBossBattleCustom; re
 BossBattleCustomData a (LNBTTagCompound;)V load
 BossBattleCustomData b ()Ljava/util/Collection; getBattles
 BossBattleCustomData c ()LNBTTagCompound; save
-BossBattleServer a (F)V setProgress
+BossBattle a (F)V setProgress
 BossBattleServer a (LEntityPlayer;)V addPlayer
 BossBattleServer a (LPacketPlayOutBoss$Action;)V sendUpdate
-BossBattleServer a (Z)LBossBattle; setDarkenSky
+BossBattle a (Z)LBossBattle; setDarkenSky
 BossBattleServer b (LEntityPlayer;)V removePlayer
-BossBattleServer b (Z)LBossBattle; setPlayMusic
-BossBattleServer c (Z)LBossBattle; setCreateFog
+BossBattle b (Z)LBossBattle; setPlayMusic
+BossBattle c (Z)LBossBattle; setCreateFog
 BossBattleServer d (Z)V setVisible
 BossBattleServer h ()Ljava/util/Collection; getPlayers
 BossBattleServer j visible
@@ -1424,7 +1424,7 @@ ChunkProviderServer a (LTicketType;LChunkCoordIntPair;ILjava/lang/Object;)V addT
 ChunkProviderServer a (Ljava/util/function/BooleanSupplier;)V tick
 ChunkProviderServer a (Z)V save
 ChunkProviderServer a playerChunkMap
-ChunkProviderServer b (II)Z isLoaded
+IChunkProvider b (II)Z isLoaded
 ChunkProviderServer b (LBlockPosition;)V flagDirty
 ChunkProviderServer b (LEntity;LPacket;)V broadcast
 ChunkProviderServer b (LTicketType;LChunkCoordIntPair;ILjava/lang/Object;)V removeTicket
@@ -2266,7 +2266,7 @@ EntityFishingHook$HookState a FLYING
 EntityFishingHook$HookState b HOOKED_IN_ENTITY
 EntityFishingHook$HookState c BOBBING
 EntityFox a (LEntityFox$Type;)V setFoxType
-EntityFox bo ()Z isCrouching
+Entity bo ()Z isCrouching
 EntityFox by FIRST_TRUSTED_PLAYER
 EntityFox bz SECOND_TRUSTED_PLAYER
 EntityFox eD ()V initializePathFinderGoals
@@ -2673,7 +2673,7 @@ EntityPositionTypes$Surface a ON_GROUND
 EntityPositionTypes$Surface b IN_WATER
 EntityPositionTypes$Surface c NO_RESTRICTIONS
 EntityPotion b (LItemStack;)V setItem
-EntityPotion f ()LItemStack; getItem
+EntityPotion f ()LItemStack; getStack
 EntityPotion i ()V splash
 EntityPotion k ()Z isLingering
 EntityProjectile ao shooterId
@@ -2879,7 +2879,7 @@ EntityVex r (Z)V setCharging
 EntityVillager a (LVillagerData;)V setVillagerData
 EntityVillager eI ()Z canBreed
 EntityVillager eX ()V populateTrades
-EntityVillager u (I)V setExperience
+EntityVillager u (I)V setXp
 EntityVillagerAbstract bw trades
 EntityVillagerAbstract by tradingPlayer
 EntityVillagerAbstract bz inventory
@@ -3378,7 +3378,7 @@ IEntityAccess a (LEntity;LAxisAlignedBB;Ljava/util/function/Predicate;)Ljava/uti
 IFluidContainer a (LGeneratorAccess;LBlockPosition;LIBlockData;LFluid;)Z place
 IFluidContainer a (LIBlockAccess;LBlockPosition;LIBlockData;LFluidType;)Z canPlace
 IFluidSource b (LGeneratorAccess;LBlockPosition;LIBlockData;)LFluidType; removeFluid
-IHopper w ()LWorld; getWorld
+IHopper w ()LWorld; getHopperWorld
 IInventory L_ ()I getMaxStackSize
 IInventory O_ ()I getSize
 IInventory a (I)LItemStack; getItem
@@ -3393,7 +3393,7 @@ ILightAccess l ()LIBlockAccess; getWorld
 IMaterial i ()LItem; getItem
 IMerchant a (LEntityHuman;LIChatBaseComponent;I)V openTrade
 IMerchant e (LEntityHuman;)V setTradingPlayer
-IMerchant eB ()LWorld; getWorld
+IMerchant eB ()LWorld; getTraderWorld
 IMerchant er ()I getExperience
 IMerchant es ()LEntityHuman; getTrader
 IMerchant eu ()LMerchantRecipeList; getOffers
@@ -4323,7 +4323,7 @@ NBTTagInt b data
 NBTTagIntArray b data
 NBTTagIntArray g ()[I getInts
 NBTTagList a (I)LNBTTagCompound; getCompound
-NBTTagList c (I)LNBTBase; remove
+NBTList c (I)LNBTBase; remove
 NBTTagList c list
 NBTTagList h type
 NBTTagList j (I)Ljava/lang/String; getString
