From bd854f408bee3b4b9a0ef3ff59005945bb29a27d Mon Sep 17 00:00:00 2001
From: Ramid Khan <ramidzkh@gmail.com>
Date: Tue, 14 Jan 2020 16:35:03 +1100
Subject: [PATCH] Fix 1.15.1 mappings

Signed-off-by: Ramid Khan <ramidzkh@gmail.com>
---
 mappings/bukkit-1.15.1-members.csrg | 19 +++++++++----------
 1 file changed, 9 insertions(+), 10 deletions(-)

diff --git a/mappings/bukkit-1.15.1-members.csrg b/mappings/bukkit-1.15.1-members.csrg
index 0139b88..5a43c45 100644
--- a/mappings/bukkit-1.15.1-members.csrg
+++ b/mappings/bukkit-1.15.1-members.csrg
@@ -1261,13 +1261,13 @@ BossBattleCustomData a (LMinecraftKey;LIChatBaseComponent;)LBossBattleCustom; re
 BossBattleCustomData a (LNBTTagCompound;)V load
 BossBattleCustomData b ()Ljava/util/Collection; getBattles
 BossBattleCustomData c ()LNBTTagCompound; save
-BossBattleServer a (F)V setProgress
+BossBattle a (F)V setProgress
 BossBattleServer a (LEntityPlayer;)V addPlayer
 BossBattleServer a (LPacketPlayOutBoss$Action;)V sendUpdate
-BossBattleServer a (Z)LBossBattle; setDarkenSky
+BossBattle a (Z)LBossBattle; setDarkenSky
 BossBattleServer b (LEntityPlayer;)V removePlayer
-BossBattleServer b (Z)LBossBattle; setPlayMusic
-BossBattleServer c (Z)LBossBattle; setCreateFog
+BossBattle b (Z)LBossBattle; setPlayMusic
+BossBattle c (Z)LBossBattle; setCreateFog
 BossBattleServer d (Z)V setVisible
 BossBattleServer h ()Ljava/util/Collection; getPlayers
 BossBattleServer j visible
@@ -1403,7 +1403,7 @@ ChunkProviderServer a (LTicketType;LChunkCoordIntPair;ILjava/lang/Object;)V addT
 ChunkProviderServer a (Ljava/util/function/BooleanSupplier;)V tick
 ChunkProviderServer a (Z)V save
 ChunkProviderServer a playerChunkMap
-ChunkProviderServer b (II)Z isLoaded
+IChunkProvider b (II)Z isLoaded
 ChunkProviderServer b (LBlockPosition;)V flagDirty
 ChunkProviderServer b (LEntity;LPacket;)V broadcast
 ChunkProviderServer b (LTicketType;LChunkCoordIntPair;ILjava/lang/Object;)V removeTicket
@@ -2220,7 +2220,7 @@ EntityFishingHook$HookState a FLYING
 EntityFishingHook$HookState b HOOKED_IN_ENTITY
 EntityFishingHook$HookState c BOBBING
 EntityFox a (LEntityFox$Type;)V setFoxType
-EntityFox bo ()Z isCrouching
+Entity bo ()Z isCrouching
 EntityFox eD ()V initializePathFinderGoals
 EntityFox eq ()LEntityFox$Type; getFoxType
 EntityFox er ()Z isSitting
@@ -2825,7 +2825,6 @@ EntityVex r (Z)V setCharging
 EntityVillager a (LVillagerData;)V setVillagerData
 EntityVillager eI ()Z canBreed
 EntityVillager eX ()V populateTrades
-EntityVillager u (I)V setExperience
 EntityVillagerAbstract bw trades
 EntityVillagerAbstract by tradingPlayer
 EntityVillagerAbstract bz inventory
@@ -3322,7 +3321,7 @@ IEntityAccess a (LEntity;LAxisAlignedBB;Ljava/util/function/Predicate;)Ljava/uti
 IFluidContainer a (LGeneratorAccess;LBlockPosition;LIBlockData;LFluid;)Z place
 IFluidContainer a (LIBlockAccess;LBlockPosition;LIBlockData;LFluidType;)Z canPlace
 IFluidSource b (LGeneratorAccess;LBlockPosition;LIBlockData;)LFluidType; removeFluid
-IHopper w ()LWorld; getWorld
+IHopper w ()LWorld; getHopperWorld
 IInventory L_ ()I getMaxStackSize
 IInventory O_ ()I getSize
 IInventory a (I)LItemStack; getItem
@@ -3337,7 +3336,7 @@ ILightAccess l ()LIBlockAccess; getWorld
 IMaterial i ()LItem; getItem
 IMerchant a (LEntityHuman;LIChatBaseComponent;I)V openTrade
 IMerchant e (LEntityHuman;)V setTradingPlayer
-IMerchant eB ()LWorld; getWorld
+IMerchant eB ()LWorld; getTraderWorld
 IMerchant er ()I getExperience
 IMerchant es ()LEntityHuman; getTrader
 IMerchant eu ()LMerchantRecipeList; getOffers
@@ -4257,7 +4256,7 @@ NBTTagInt b data
 NBTTagIntArray b data
 NBTTagIntArray g ()[I getInts
 NBTTagList a (I)LNBTTagCompound; getCompound
-NBTTagList c (I)LNBTBase; remove
+NBTList c (I)LNBTBase; remove
 NBTTagList c list
 NBTTagList h type
 NBTTagList j (I)Ljava/lang/String; getString
-- 
